src/solver/conv_winoRxS.cpp

```diff
@@ -191,31 +191,42 @@ inline bool IsShaderContraintsMet(const int R,
              params.problem.GetBackwardPadH() < std::pow(2, 16)))
             return false;
     }
-    const auto grid_workgroup_count_x = params.GetStream().GetMaxHardwareComputeUnits();
     if(!params.problem.IsLayoutDefault())
     {
         return false;
     }
 
+    uint32_t o_K_stride      = OH * OW;
+    uint32_t o_N_stride      = o_K_stride * K;
+    uint32_t o_N_stride_OHOW = o_N_stride + o_K_stride;
+
+    uint32_t d_C_stride    = H * W;
+    uint32_t d_N_stride    = d_C_stride * C;
+    uint32_t d_N_stride_HW = d_N_stride + d_C_stride;
```

Comment:
[Q] Are you sure that it will fit into 32 bits? Especially `o_N_stride_OHOW`?

Reply:
Thanks! Done in [7d49e9e](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1781/commits/7d49e9ec72823039f5f01067b19ee689e56ff93f)